src/batch-submitter/batch-submitter.ts

```diff
@@ -11,7 +10,7 @@ import { getContractFactory } from '@eth-optimism/contracts'
 
 /* Internal Imports */
 import { Address, Bytes32 } from '../coders'
-
+import { gasInGwei } from '../utils'
```

Comment:
Creating this helper function is a good idea, there is also an ethers helper function that does this too - https://docs.ethers.io/v5/api/utils/display-logic/

Reply:
migrating to this to minimize code we maintain!